forwardproxy_test.go

```diff
@@ -19,17 +19,18 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
-	_ "github.com/mholt/caddy/caddyhttp/header"
-	_ "github.com/mholt/caddy/caddyhttp/httpserver"
-	_ "github.com/mholt/caddy/caddyhttp/redirect"
-	_ "github.com/mholt/caddy/caddyhttp/root"
 	"io"
 	"net"
 	"net/http"
 	"net/url"
 	"strings"
 	"testing"
 	"time"
+
+	_ "github.com/mholt/caddy/caddyhttp/header"
```

Comment:
why move these around?

Reply:
`goimports` puts third-party packages in their own little sections at the end of the imports to keep it tidy.

Reply:
Hah, somehow I never heard of `goimports`. Alright then, thanks.